src/pages/rayong/Login/Login.js

```diff
@@ -1,7 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../Login/Login.scss';
 
 function LoginRa() {
-  return <h1>hello world</h1>;
+  const navigate = useNavigate();
+  const [inputValues, setInputValues] = useState({
+    id: '',
+    pw: '',
+  });
+  const handleInput = event => {
```

Comment:
```suggestion
  });
  
  const handleInput = event => {
```

이 부분도 가독성을 위해 한칸 띄워주는 것도 좋겠습니다.